src/App.tsx

```diff
@@ -1,8 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
```

Comment:
```suggestion
  state = {state: Readonly<State> = {
```
можна додати такий тип для більшого захисту)